test/integration/contract.js

```diff
@@ -217,26 +217,26 @@ describe('Contract', function () {
       const { salt: _salt } = await contract.aensPreclaim(name)
       // @TODO enable after next HF
       // const commitmentId = commitmentHash(name, _salt)
-      const preclaimSig = await contract.delegateNamePreclaimSignature(contractAddress, { onAccount: current })
+      const preclaimSig = await contract.createAensDelegationSignature(contractAddress, { onAccount: current })
```

Comment:
no need to distinguish between different signatures here in this test. currently we have:
- preclaimSig
- claimSig
- transferSig
- revokeSig

after your changes we only need following signatures in the test:
- preclaimSig
- aensDelegationSig

Reply:
Done. `revokeSig` is a different signature as it needs to be signed by the new owner once the transfer has occurred.
So we do have 3 signatures
* preclaimSig
* aensDelegationSig
* revokeSig